aea/cli/common.py

```diff
@@ -383,8 +378,10 @@ def convert(self, value, param, ctx):
             # everything ok - return the parameter to the command
             return value
         except Exception:
-            logger.error("The name provided is not a path to an AEA project.")
-            self.fail(value, param, ctx)
+            # self.fail(value, param, ctx)
```

Comment:
do we still need self.fail? if so, remove commenting, else remove comment altogether.